src/term/mod.rs

```diff
@@ -950,12 +954,16 @@ impl ansi::Handler for Term {
 
     #[inline]
     fn save_cursor_position(&mut self) {
-        err_println!("[unimplemented] save_cursor_position");
+        self.saved_cursor = Some(self.cursor);
     }
 
     #[inline]
     fn restore_cursor_position(&mut self) {
-        err_println!("[unimplemented] restore_cursor_position");
+        if let Some(pos) = self.saved_cursor {
+            self.cursor = pos;
```

Comment:
> SIGWINCH would be for programs running inside Alacritty.

I realize that; I ask because I noticed that Heatseeker's usual `SIGWINCH` handling doesn't seem to be kicking in in response to the Alacritty window being resized. It stands to reason that the `TIOCSWINSZ` ioctl should trigger `SIGWINCH`, so I'll investigate that a bit more.